src/taipy/gui/gui.pyi

```diff
@@ -208,6 +213,10 @@ class Gui:
     def __bind_local_func(self, name: str): ...
     def load_config(self, config: Config) -> None: ...
     def broadcast(self, name: str, value: t.Any): ...
+    def _broadcast_all_clients(self, name: str, value: t.Any): ...
+    def _set_broadcast(self, broadcast: bool = ...): ...
+    def _del_broadcast(self) -> None: ...
+    def _is_broadcasting(self) -> bool: ...
```

Comment:
do we need to make those available in gui.pyi ?

Reply:
a script generates that. we can optionally remove those but I think it is ok to keep it. Pyi should be kept as an alias to the original file